operator/v1/types.go

```diff
@@ -53,6 +53,13 @@ type ClusterManagerSpec struct {
 	// +optional
 	// +kubebuilder:default={mode: Default}
 	DeployOption ClusterManagerDeployOption `json:"deployOption,omitempty"`
+
+	// FeatureGates represents the list of feature gates for hub registration
+	// If it is set empty, default feature gates will be used.
+	// If it is set not empty, hub registration controller will use all items of it as args.
```

Comment:
I do not think we should restrict it to registration right? Also we should explicitly describe this is to override the default features.

Reply:
If we are not restrict it to registration, we do not know paas items of it to which deployment(registration? work? placement? or their webhook?).

Reply:
I still think we should have a FeatureGates, and we probably should make it as a struct instead of a []string. Alternatively we should have something like:

```
registrationConfig:
  featureGates: [...]
```

Reply:
I think you are right. We need an entry that contains all registration configuration.